src/components/schedule/worker-info-section/shifts-section/shift-cell/shift-cell.component.tsx

```diff
@@ -16,7 +16,6 @@ import { MouseEventListener } from "../../../../common-components/mouse-event-li
 import { ShiftCellContent } from "./shift-cell-content.component";
 import { ShiftsCellDropdown } from "./shifts-cell-dropdown.component";
 
-// TODO: Create a config file, that will contain this definitions
```

Comment:
Usunąłem. Raczej nie jest to teraz istotne.
Jeśli pojawi się taka potrzeba, to wtedy będziemy wprowadzać